tests/spread/integration/libc-bin/task.yaml

```
@@ -1,6 +1,17 @@
 summary: Integration tests for libc-bin
 
+environment:
+  SLICE/getconf: "getconf"
+  SLICE/getent: "getent"
+
 execute: |
-  rootfs="$(install-slices libc-bin_getconf)"
+  rootfs="$(install-slices libc-bin_${SLICE} )"
 
-  chroot "$rootfs" getconf -a
+  case SLICE in
+    getconf)
+      chroot "$rootfs" getconf -a
+    ;;
+    getent)
+      chroot "$rootfs" getent --version | grep -q getent
```

It seems we can also do a real run `getent hosts` in the chrooted environment. It should exit with code 0.

---

Yeah, let's do at least this. Maybe check that it can get a host from `/etc/hosts` too:

```suggestion
      chroot "$rootfs" getent --version | grep -q getent
      chroot "${rootfs}" getent hosts
      ! chroot "${rootfs}" getent hosts localhost
      
      mkdir "${rootfs}/etc"
      echo "::1 localhost" > "${rootfs}/etc/hosts
      chroot "${rootfs}" getent hosts localhost
```